vue-components/src/components/Checkbox.vue

```diff
@@ -53,11 +51,14 @@ export default Vue.extend( {
 $label: '.wikit-checkbox__label';
 
 .wikit-checkbox {
+	display: flex;
+
 	&__label {
 		@include Label;
-
 		display: flex;
 		align-items: center;
+		word-break: break-all;
+		margin-inline-end: $dimension-layout-xxsmall;
```

Comment:
```suggestion
		margin-inline-end: $dimension-spacing-small;
```

Reply:
Agreeing with @micgro42's comment, since the spacing is applied at a component level.